scripts/isolate_tests.py

```diff
@@ -91,5 +91,8 @@ def write_cases(tests):
             if docs:
               cases = extract_docs_cases(path)
             else:
-              cases = extract_test_cases(path)
+              if f.endswith(".sol"):
+                write_cases([open(path, "r").read()])
+              else:
+                cases = extract_test_cases(path)
             write_cases(cases)
```

Comment:
There is quite a number of constracts in test/compilationTests/ already - so far it looks to me that they had not been processed in isolate_tests.py, but they are now.
I haven't found the place where test/compilationTests/ is processed so far - it may be that this needs adjustment to avoid double-testing those scripts.

Reply:
Those are run via `cmdlineTests.sh`, and they are actually multi-file codebases, so it does not make sense to extract them into individual files, because the import each other